substrate/frame/revive/src/weights.rs

```diff
@@ -1388,116 +1392,123 @@ impl WeightInfo for () {
 		//  Estimated: `0`
-		// Minimum execution time: 1_910_000 picoseconds.
-		Weight::from_parts(2_205_396, 0)
-			// Standard Error: 2
-			.saturating_add(Weight::from_parts(538, 0).saturating_mul(n.into()))
+		// Minimum execution time: 2_005_000 picoseconds.
+		Weight::from_parts(2_225_035, 0)
+			// Standard Error: 1
+			.saturating_add(Weight::from_parts(526, 0).saturating_mul(n.into()))
 			.saturating_add(RocksDbWeight::get().writes(1_u64))
 	}
 	fn seal_value_transferred() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 224_000 picoseconds.
-		Weight::from_parts(274_000, 0)
+		// Minimum execution time: 263_000 picoseconds.
+		Weight::from_parts(304_000, 0)
 	}
 	fn seal_minimum_balance() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 231_000 picoseconds.
-		Weight::from_parts(279_000, 0)
+		// Minimum execution time: 258_000 picoseconds.
+		Weight::from_parts(307_000, 0)
 	}
 	fn seal_return_data_size() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 229_000 picoseconds.
-		Weight::from_parts(267_000, 0)
+		// Minimum execution time: 272_000 picoseconds.
+		Weight::from_parts(304_000, 0)
 	}
 	fn seal_call_data_size() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 218_000 picoseconds.
-		Weight::from_parts(267_000, 0)
+		// Minimum execution time: 280_000 picoseconds.
+		Weight::from_parts(306_000, 0)
 	}
 	fn seal_gas_limit() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 225_000 picoseconds.
-		Weight::from_parts(280_000, 0)
+		// Minimum execution time: 317_000 picoseconds.
+		Weight::from_parts(339_000, 0)
 	}
 	fn seal_gas_price() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 274_000 picoseconds.
-		Weight::from_parts(323_000, 0)
+		// Minimum execution time: 261_000 picoseconds.
+		Weight::from_parts(298_000, 0)
 	}
 	fn seal_base_fee() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 239_000 picoseconds.
-		Weight::from_parts(290_000, 0)
+		// Minimum execution time: 247_000 picoseconds.
+		Weight::from_parts(279_000, 0)
 	}
 	fn seal_block_number() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 224_000 picoseconds.
-		Weight::from_parts(274_000, 0)
+		// Minimum execution time: 270_000 picoseconds.
+		Weight::from_parts(309_000, 0)
+	}
+	fn seal_block_author() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 776_000 picoseconds.
+		Weight::from_parts(879_000, 0)
```

Comment:
No PoV impact. I assume reading this item is free since it is per block (never written to storage). But could also bee that it is just empty during the benchmark?

Reply:
Aye